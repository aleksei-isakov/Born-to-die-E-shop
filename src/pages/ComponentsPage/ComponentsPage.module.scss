.title {
  font-size: 32px;
  margin: 25px 0;
}

.sub-title {
  font-size: 20px;
  padding: 5px 0;
}
