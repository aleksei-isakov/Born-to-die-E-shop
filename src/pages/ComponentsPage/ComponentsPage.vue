<template>
  <div>
    <div>
      <div class="title">Buttons</div>
      <div>
        <div class="sub-title">Flat</div>
        <md-button>Default</md-button>
        <md-button class="md-primary">Primary</md-button>
        <md-button class="md-accent">Accent</md-button>
        <md-button disabled>Disabled</md-button>
      </div>

      <div>
        <div class="sub-title">Raised</div>
        <md-button class="md-raised">Default</md-button>
        <md-button class="md-raised md-primary">Primary</md-button>
        <md-button class="md-raised md-accent">Accent</md-button>
        <md-button class="md-raised" disabled>Disabled</md-button>
      </div>
    </div>
    <div>
      <div class="title">Tabs</div>
      <md-tabs class="md-transparent" md-alignment="fixed">
        <md-tab id="tab-1" md-label="Sign In"></md-tab>
        <md-tab id="tab-2" md-label="Sign Up"></md-tab>
      </md-tabs>
    </div>
  </div>
</template>

<script>
import Vue from 'vue';
import VueMaterial from 'vue-material';

Vue.use(VueMaterial);

export default {
  name: 'ComponentsPage'
};
</script>

<style>
@import './ComponentsPage.module.scss';
</style>
