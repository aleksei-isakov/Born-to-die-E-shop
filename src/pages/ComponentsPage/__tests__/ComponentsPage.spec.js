import { shallowMount } from '@vue/test-utils';
import ComponentsPage from '../ComponentsPage.vue';

describe('ComponentsPage', () => {
  it('renders a valid snapshot', () => {
    const wrapper = shallowMount(ComponentsPage);

    expect(wrapper).toMatchSnapshot();
  });
});
