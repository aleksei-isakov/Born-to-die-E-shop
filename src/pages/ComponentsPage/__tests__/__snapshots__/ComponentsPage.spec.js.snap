// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentsPage renders a valid snapshot 1`] = `
<div>
  <div>
    <div class="title">Buttons</div>
    <div>
      <div class="sub-title">Flat</div>
      <md-button-stub mdripple="true" type="button">Default</md-button-stub>
      <md-button-stub mdripple="true" type="button" class="md-primary">Primary</md-button-stub>
      <md-button-stub mdripple="true" type="button" class="md-accent">Accent</md-button-stub>
      <md-button-stub mdripple="true" type="button" disabled="true">Disabled</md-button-stub>
    </div>
    <div>
      <div class="sub-title">Raised</div>
      <md-button-stub mdripple="true" type="button" class="md-raised">Default</md-button-stub>
      <md-button-stub mdripple="true" type="button" class="md-raised md-primary">Primary</md-button-stub>
      <md-button-stub mdripple="true" type="button" class="md-raised md-accent">Accent</md-button-stub>
      <md-button-stub mdripple="true" type="button" disabled="true" class="md-raised">Disabled</md-button-stub>
    </div>
  </div>
  <div>
    <div class="title">Tabs</div>
    <md-tabs-stub mdswipethreshold="150" mdswiperestraint="100" mdswipetime="300" mdalignment="fixed" mdelevation="0" class="md-transparent">
      <md-tab-stub id="tab-1" mdlabel="Sign In" mdtemplatedata="[object Object]"></md-tab-stub>
      <md-tab-stub id="tab-2" mdlabel="Sign Up" mdtemplatedata="[object Object]"></md-tab-stub>
    </md-tabs-stub>
  </div>
</div>
`;
